script/Arcueid/Day 12/12_00_ARC12_2.txt

```diff
@@ -414,17 +414,18 @@ I don't think I've ever heard something so viscerally repulsive before.
 [sha:f23d5072a1916c81a6fadfa3d46df665b38c0a22]{
 -- Page 58, Offset 30676.
 -- 「“本は片手に掴めるほど小さく、
-"A book so small, you could grasp it in one hand.
+"Books are small enough to grasp in one hand.
 }
 [sha:ec42bc937721a6c35ac56e991ebd3175cb9dfa8f]{
 -- Page 58, Offset 30677.
 -- 紙は年月に霞むほど<脆|もろ>い。
-Pages as muddled and vague as the passage of time itself.
+//I remember we discussed this line in the GPR, and I suggested the previous interpretation, but re-reading this scene I realize that it's actually talking generally. Just as the previous line. Suddenly it is much easier to make sense of.
+Their pages are brittle enough to fade with the passage of time.
```

Comment:
Pages being so brittle as to _fade_ reads a bit off to me still. _Ink_ fades. Pages can yellow, but that's not a consequence of being brittle. I think I'd originally tweaked this to 
> Their pages so brittle that they crumble with the passage of time.

just so that there's agreement

Reply:
Sure